src/posts/writing-agnostic-services/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Developing technology-agnostic services"
+description: ""
+date: 2024-06-25
+thumb: "edge2edge-media-qHhJkxare7A-unsplash.jpg"
+thumbCaption: "Credit: Edge2Edge Media via Unsplash"
+thumbCaptionHref: "edge2edge-media-qHhJkxare7A-unsplash.jpg"
+tags: 
+    - architecture
+    - cloud
+    - kubernetes
+    - EU-DORA
+---
+
+
+We (at factor10) have been discussing the EU's Digital Operational Resilience Act lately,
+what it means, and what impact it might have.
+
+What we have yet to discuss, though, is _how on earth_ you are supposed to comply with it.
+I'm not going to do that here either,
+but I want to address how you can develop software so you can easily move it between providers or swap out specific technologies,
+which is one piece of that puzzle.
+
+---
+
+If your immediate reaction was: "That's easy, you just use Kubernetes",
```

Comment:
VIlken oväntad "immediate reaction" tänker jag... :D Jag trodde den direkta reaktionen skulle vara något i stil med att håll det hela supersimpelt så går det att flytta med minimal ansträngning. :)

Reply:
Jag tror att den är rätt vanlig. "Cloud Native", "Operating system of the cloud" och allt sånt där.

Man jag spår ju bara, och ville få in att Kubernetes inte löser alla problem :)